client/src/assets/icon-heart-empty.svg

```diff
@@ -0,0 +1,3 @@
+<svg id="icon-heart-empty-green" xmlns="http://www.w3.org/2000/svg" width="35" height="33.675" viewBox="0 0 35 33.675">
+  <path id="icon-heart-empty-green-2" data-name="icon-heart-empty-green" d="M17.5,33.675C8.09,26.951,3.155,20.834,1.131,15.715-4.29,2,11.2-4.565,17.5,3.533,23.8-4.565,39.29,2,33.869,15.715,31.845,20.834,26.91,26.951,17.5,33.675Z" fill="rgba(51,51,51,0.5)"/>
+</svg>
```

Comment:
You can reduce the file size by removing unused fields like `id` or `data-name`.
These tools might be helpful:
https://www.svgminify.com/
https://imageoptim.com/

Reply:
Fixed in commit https://github.com/ncbecker/podium/pull/42/commits/98fe3499b42fecbebde58e42a53fa256cd6e26d5
Thanks 👍